sr/comp/mixtape/mixtape.py

```diff
@@ -196,5 +201,5 @@ def generate_play_actions(
             else:
                 raise ValueError(f"Unknown track type at index {idx} start:{track['start']}")
 
-            print('Scheduling', name, 'for', track['start'])
+            logging.debug(F"Scheduling {name} for {track['start']}")
```

Comment:
nit (I didn't even realise that `F` strings were a thing)
```suggestion
            logging.debug(f"Scheduling {name} for {track['start']}")
```